libs/sql-parser/src/main/java/io/crate/sql/tree/Explain.java

```diff
@@ -27,12 +27,22 @@
 
 public class Explain extends Statement {
 
+    public enum Option {
+        ANALYZE,
+        COSTS,
+        NONE
+    }
+
     private final Statement statement;
+    private final Option option;
     private final boolean analyze;
+    private final boolean activated;
 
-    public Explain(Statement statement, boolean analyze) {
+    public Explain(Statement statement, boolean analyze, Option option, boolean activated) {
```

Comment:
We could have split this into two statements. One for analyze and one for the options. However, this squeezes all into one statement and converts `EXPLAIN ANALYZE` into `EXPLAIN (ANALYZE TRUE)` internally. Later only the options are processed.